src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -370,13 +370,7 @@ public bool SupportsTransactions
             get { return _supportsTransactions; }
             set
             {
-#if !CORECLR
                 _supportsTransactions = value;
-#else
-                // Disable 'SupportsTransactions' in CoreCLR
-                // No transaction supported on CSS due to the lack of System.Transactions namespace
-                _supportsTransactions = false;
-#endif
```

Comment:
Let's keep the transaction related if/defs for now. More work is needed to enable transaction support.